msg/SensorOpticalFlow.msg

```diff
@@ -3,9 +3,9 @@ uint64 timestamp_sample
 
 uint32 device_id               # unique device ID for the sensor that does not change between power cycles
 
-float32[2] pixel_flow          # (radians) optical flow in radians where a positive value is produced by a RH rotation about the body axis
+float32[2] pixel_flow          # (radians) optical flow in radians where a negative value is produced by a RH rotation about the body axis
```

Comment:
```suggestion
float32[2] pixel_flow          # (radians) optical flow in radians where a positive value is produced by a RH rotation of the sensor about the body axis
```

> EKF assumes positive LOS rate is produced by a RH rotation of the image about the sensor axis.

The opposite of this is a negative rotation of the image, or a positive rotation of the sensor. This message tells that a positive pixel flow is produces by a positive rotation of the sensor, which is correct (maybe just modify to the "RH rotation **of the sensor** about the body axis")

Reply:
If it were positive rotation wouldn't it have to be from the features perspective? From the sensors perspective (camera looking down) the drone moving forward causes the features to move backwards in the image (-X). This looks like the features are rotating in the negative direction around the Y axis.

Reply:
Yes, the features move backwards in the image, but this results in a positive value.

![image](https://github.com/user-attachments/assets/744da908-8f33-46a0-91bc-c650408978f4)
(ref.: https://docs.px4.io/main/en/sensor/optical_flow.html#setup)

In the EKF we talk about the direction of the features instead of the rotation of the sensor, and this is why the direction is opposite.

Reply:
> a positive rotation of the sensor

It just clicked. If nothing moves and the vehicle pitches in the positive direction, the features will move backwards in the image. Therefore flow and gyro have the same sign and are proportional. Thanks for bearing with me.

A more intuitive way to think about this is the flow data being in pixel coordinate system. But of course we have to keep the units in radians.